Core/Core/Files/FileUploadContext.swift

```diff
@@ -20,32 +20,28 @@ import Foundation
 
 public enum FileUploadContext: Codable {
     enum CodingKeys: String, CodingKey {
-        case type, userID, courseID, assignmentID, comment
+        case type, context, courseID, assignmentID, comment
     }
 
     enum Key: String, Codable {
-        case course, user, submission, submissionComment
+        case context, submission, submissionComment
     }
 
-    case course(String)
-    case user(String)
+    case context(Context)
```

Comment:
The RN version of the reply screen had the rce upload to context/contextID/files, which could be a group not already possible in this enum, so I made it more generic.